src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -56,11 +81,19 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
         Entry lastEntry = model.getFilteredExpenditureList().get(indexLastEntry.getZeroBased());
 
         ExpenditureBuilder expenditureInList = new ExpenditureBuilder(lastEntry);
-        Entry editedEntry = expenditureInList.withDescription(VALID_DESC_TUITION)
-                .withAmount(VALID_AMT_TUITION).withTag(VALID_TAG_PERSONAL).build();
 
-        EditEntryDescriptor descriptor = new EditEntryDescriptorBuilder().withType(VALID_TYPE_EXPENDITURE)
-                .withDescription(VALID_DESC_TUITION).withAmount(VALID_AMT_TUITION).withTag(VALID_TAG_PERSONAL).build();
+        Entry editedEntry = expenditureInList
+                .withDescription(VALID_DESC_TUITION)
+                .withAmount(VALID_AMT_TUITION)
+                .build();
+
+        // Specify only the description and amount field to be updated
```

Comment:
Similarly, this test should be checking for some fields only, so I specified description and amount.